wrappers/python/README.md

```diff
@@ -6,27 +6,62 @@
 ## Installation
 
 ```bash
-pip install zxing-cpp
+pip install --upgrade zxing-cpp
 ```
+
 or
 
 ```bash
 python setup.py install
 ```
 
-[Note: To install via `setup.py` (or via `pip install` in case there is no pre-build wheel available for your platfor or python version), you need a suitable [build environment](https://github.com/zxing-cpp/zxing-cpp#build-instructions) including a c++ compiler.]
+Note: To install via `setup.py` (or via `pip install` in case there is no pre-build wheel available for your platform or python version), you need a suitable [build environment](https://github.com/zxing-cpp/zxing-cpp#build-instructions) including a C++ compiler.
 
 ## Usage
 
+For the examples, you need to install Pillow (for PIL) and/or CV2 with:
+
+```bash
+pip install --upgrade Pillow opencv-python
+```
+
+An example for writing a code to a PNG file with the help of PIL is:
+
 ```python
-import cv2
+import sys
 import zxingcpp
+from PIL import Image
+
+if len(sys.argv) < 3:
+	img = zxingcpp.write_barcode(zxingcpp.BarcodeFormat.QRCode,
+		'I have the best words.', width=200, height=200)
+else:
+	img = zxingcpp.zxingcwrite_barcode(zxingcpp.barcode_format_from_str(sys.argv[1]),
```

Comment:
`zxingcwrite_barcode` method does not exists. To make this as simple as possible, please leave only the first `if` branch.